components/Tabs/src/ColorThemer/MDCTabBarColorThemer.h

```diff
@@ -22,10 +22,9 @@
  details on replacement APIs.
  Learn more at docs/theming.md#migration-guide-themers-to-theming-extensions
  */
-@interface MDCTabBarColorThemer : NSObject
-@end
-
-@interface MDCTabBarColorThemer (ToBeDeprecated)
+__deprecated_msg("Please use the Theming extension instead.") @interface MDCTabBarColorThemer
```

Comment:
Tip: Please consider pointing the user to the correct target or category. They may know what a "Theming extension" is by that name.